build/includes/windows.mk

```diff
@@ -31,20 +31,6 @@ sha_dir = $(shell sha256sum  $(1) | cut -d' ' -f1 | sha256sum | head -c 10 )
 
 # Minikube executable
 MINIKUBE ?= minikube.exe
-# Default minikube driver
-MINIKUBE_DRIVER ?= hyperv
-# set docker env for minikube
-MINIKUBE_DOCKER_ENV ?= eval $$($(MINIKUBE) docker-env --shell=bash) && \
- export DOCKER_CERT_PATH=$$(echo $$DOCKER_CERT_PATH | $(win_to_wsl_path))
-
-# minikube shell mount for certificates
-minikube_cert_mount = $(cert_path):$(cert_path)
```

Comment:
I think this one should stay there

Reply:
So the previous Windows support was super hacky, brittle and I expect it doesn't work anymore.

We were rewriting windows path to linux paths in make files. If we decide to continue supporting it, it needs a whole new rewrite. So I've removed it for now. If we need to add new os specific tooling, then lets revisit then. Right now, between OSX and Linux, it's all the same.